client/src/Header.js

```diff
@@ -1,16 +1,42 @@
 import Logo from "./assets/images/CTY-logo-rectangle.png";
 import { NavLink } from "react-router-dom";
+import { useNavigate } from "react-router-dom";
 import { useState, useEffect } from "react";
 import "./Header.css";
 
 const Header = () => {
 	const [role, setRole] = useState(null);
+	const navigate = useNavigate();
 
 	useEffect(() => {
 		const userType = localStorage.getItem("userType");
 		setRole(userType);
 	}, []);
 
+	const handleLogout = async () => {
+		try {
+			const token = localStorage.getItem("token");
+
+			const response = await fetch("/logout", {
```

Comment:
This endpoint currently doesn't exist. Can you please create it and make it so it deletes all sessions for that user?